public/index.php

```diff
@@ -16,7 +16,7 @@
  * @since 0.1
  */
 
-session_start();
```

Comment:
This needs to be here since you have to start the session every time a page loads.

Reply:
I've replaced this with a function that checks to see if there's already an active session before starting a new one. This way, when the session variables I'm using for the timeout flag aren't being reset.

```
// Only create a session if there is not already one there
if (session_status() === PHP_SESSION_NONE){
    session_start();
}
```